karspexet/settings.py

```diff
@@ -154,3 +187,81 @@
 
 EMAIL_BACKEND = ENV.get("email_backend", 'django.core.mail.backends.smtp.EmailBackend')
 PAYMENT_PROCESS = ENV.get("payment_process", "not set")
+SITE_ID = 1
+CMS_TEMPLATES = [
+    ('home.html', 'Home page template'),
+    ('about.html', 'Content page template'),
+]
+LANGUAGES = [
+    ('en', 'English'),
+    ('sv', 'Swedish'),
+]
+THUMBNAIL_HIGH_RESOLUTION = True
+
+THUMBNAIL_PROCESSORS = (
+    'easy_thumbnails.processors.colorspace',
+    'easy_thumbnails.processors.autocrop',
+    'filer.thumbnail_processors.scale_and_crop_with_subject_location',
+    'easy_thumbnails.processors.filters'
+)
+
+FILER_STORAGES = {
+    'public': {
+        'main': {
+            'ENGINE': 'filer.storage.PublicFileSystemStorage',
+            'OPTIONS': {
+                'location': 'filer_public',
+                'base_url': '/static/',
+            },
+            'UPLOAD_TO': 'filer.utils.generate_filename.randomized',
+            'UPLOAD_TO_PREFIX': 'filer_public',
+        },
+        'thumbnails': {
+            'ENGINE': 'filer.storage.PublicFileSystemStorage',
+            'OPTIONS': {
+                'location': 'filer_public_thumbnails',
+                'base_url': '/static/',
+            },
+        },
+    }
+}
+
+CMS_PLACEHOLDER_CONF = {
+    None: {
+        "plugins": ['TextPlugin', 'LinkPlugin'],
+        'excluded_plugins': ['InheritPlugin'],
+    },
+    'image': {
+        'plugins': ['PicturePlugin'],
+        'name': "Image",
+        'language_fallback': True,
+        'default_plugins': [
+            {
+                'plugin_type': 'TextPlugin',
+                'values': {
+                    'body':'<p>Lorem ipsum dolor sit amet...</p>',
```

Comment:
This should probably be something more helpful/real looking input.

Reply:
This is only ever used as templating text on newly generated draft pages. It will not show up in production unless someone actually publishes a CMS page with that still in it, so I think it's fine this way. 😄

Reply:
That's what I meant, something helpful should show up so that the author has an idea of what they should/could write.

"Put the body of your article here, you can use the buttons above for styling..."

Reply:
Ah, yes. That would be nice.